pkg/agent/config/node_config.go

```diff
@@ -52,8 +53,13 @@ var (
 	// - Use the virtual IP as onlink routing entry gateway in host routing entry.
 	// - Use the virtual IP as destination IP in host routing entry. It is used to forward DNATed NodePort packets
 	//   or replied SNATed Service packets back to Antrea gateway.
-	VirtualServiceIPv4 = net.ParseIP("169.254.169.253")
-	VirtualServiceIPv6 = net.ParseIP("fc01::aabb:ccdd:eeff")
+	// For Windows, VirtualServiceIPv4 is a virtual IP to perform SNAT for traffic of Windows ClusterIP Services from host.
+	// Such traffic needs to enter and exit antrea-gw0. Without OVS SNAT, the reply's destination IP
+	// is antrea-gw0 IP and it will not go through OVS pipeline anymore. It is used in Windows conntrackCommitTable
+	// table.
```

Comment:
This is not specific to Windows. I believe the existing comment and https://github.com/antrea-io/antrea/blob/8d228c16ab07c21bd3c66e581605e1ec29dd8a7a/pkg/agent/openflow/pipeline.go#L593-L596 have same information.

Reply:
Removed.